CHANGES.rst

```diff
@@ -953,6 +953,11 @@ astropy.extern
 astropy.io.ascii
 ^^^^^^^^^^^^^^^^
 
+- Partially fixed a performance issue when reading in parallel mode. Parallel
+  reading does not work correctly and has worse performance than serial
```

Comment:
When you say "does not work correctly" that could imply it is returning the wrong answer. I did not see any statements this the answer is wrong, just that it is slow. If so, I would reword this here and elsewhere saying that "Parallel reading currently has substantially worse performance than the default serial reading, so we now ignore ...".

Reply:
Done